pkg/constants/constants.go

```diff
@@ -38,7 +38,10 @@ const (
 	KnativeServingAPIGroupName       = KnativeServingAPIGroupNamePrefix + ".dev"
 )
 
-var KServeNamespace = getEnvOrDefault("POD_NAMESPACE", "kserve")
+var (
+	KServeNamespace              = getEnvOrDefault("POD_NAMESPACE", "kserve")
+	AutoscalerConfigmapNamespace = getEnvOrDefault("CONFIG_AUTOSCALER_NAMESPACE", DefaultKnServingNamespace)
```

Comment:
nit: since we also leverage KEDA for autoscaling a more precise name would be `CONFIG_KNATIVE_AUTOSCALER_NAMESPACE`